src/components/AllProducts/AllProducts.tsx

```diff
@@ -56,18 +56,20 @@ function AllProducts() {
         />
       </div>
       <ul className={`all-product-list ${mode}`}>
-        {allItemList.map((item) => (
-          <li key={item.id}>
-            <ProductItem
-              id={item.id}
-              imageUrl={item.images[0]}
-              name={item.name}
-              price={item.price}
-              likeCount={item.favoriteCount}
-              size={size}
-            />
-          </li>
-        ))}
+        {allItemList.map(
+          ({ id, images, name, price, favoriteCount }: Product) => (
```

Comment:
allItemList가  Product[] 타입인 것을 위에 정의할 때에 vsCode 에게 알려줬기 때문에 여기는 따로 적을 필요 없을 거 같아요!
```suggestion
          ({ id, images, name, price, favoriteCount }) => (
```